lib/nitcorn/http_request.nit

```diff
@@ -82,7 +82,7 @@ class HttpRequest
 	fun int_arg(arg_name: String): nullable Int do
 		if not all_args.has_key(arg_name) then return null
 		var i = all_args[arg_name]
-		if not i.is_numeric then return null
+		if not i.is_int then return null
 		return i.to_i
```

Comment:
What about replacing this line instead to: `return i.to_n.to_i`?

`"9.5"` would give 9 instead of `null` and it would respect the current documentation.